src/config/network.config.ts

```diff
@@ -0,0 +1,35 @@
+import { Network } from "@/utils/wallet/wallet_provider";
+
+const network = process.env.NEXT_PUBLIC_NETWORK as Network;
+
+interface NetworkConfig {
+  coinName: string;
+  coinSymbol: string;
+  explorerUrl: string;
```

Comment:
```suggestion
  mempoolApiUrl: string;
```

better to be explicit here